application/modules/guest/controllers/gateways/Stripe.php

```diff
@@ -73,53 +73,102 @@ public function create_checkout_session($invoice_url_key)
      * @param  string  $checkout_session_id
      * @return void
      */
-    public function callback($checkout_session_id)
-    {
-        try {
-            $session = $this->stripe->checkout->sessions->retrieve($checkout_session_id);
-
-            //get invoice id
-            $invoice_id = $session->client_reference_id;
-
-            $this->load->model('invoices/mdl_invoices');
+public function callback($checkout_session_id)
+{
+    try {
+        // 1) Retrieve the Checkout Session from Stripe
+        $session = $this->stripe->checkout->sessions->retrieve($checkout_session_id);
 
-            //retrieve the invoice
-            $invoice = $this->mdl_invoices->where('ip_invoices.invoice_id', $invoice_id)
-                ->get()->row();
+        // 2) Determine which invoice we’re dealing with
+        $invoice_id = $session->client_reference_id;
+        $this->load->model('invoices/mdl_invoices');
+        $invoice = $this->mdl_invoices->where('ip_invoices.invoice_id', $invoice_id)
+            ->get()->row();
 
-            $this->session->set_flashdata('alert_success', sprintf(trans('online_payment_payment_successful'), $invoice->invoice_number));
+        // 3) DEBUG LOGGING (optional but recommended)
+        //    Make sure your PHP error log is set up (log_errors=On in php.ini)
+        error_log("Stripe callback reached. Checkout session id: " . $checkout_session_id);
```

Comment:
codeigniter has a function log_message() which makes the logging end up on `/application/logs/`:
```
log_message('debug', 'Logging about Stripe');
```